src/node_util.cc

```diff
@@ -16,6 +16,22 @@ using v8::Proxy;
 using v8::String;
 using v8::Value;
 
+static void GetOwnNonIndicesProperties(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Local<Context> context = env->context();
+
+  v8::Local<v8::Object> object = args[0].As<v8::Object>();
```

Comment:
I couldn't see if we have a check for this in-place. Will we crash if this fails? That said, does it ever really realistically fail?

Reply:
I added a check for that as well.